lib/Table/TableColumn.ts

```diff
@@ -130,6 +218,19 @@ export default class TableColumn {
         }
       }
 
+      if (n !== null) {
+        // If we have a `math-expression-evaluator` - use it to transform value
+        if (this.mexpPostfix) {
+          const columnPairs = this.mexpColumnPairs(i, n);
+
+          // Only transform value if all columnPairs have been set
+          if (!Object.values(columnPairs).includes(null)) {
```

Comment:
A row of a dependency may be missing, not the entire column.  
This means that we can use an expression with columns which have null values - it will just skip rows which have a null value in a dependency

Reply:
I have added a warning https://github.com/TerriaJS/terriajs/pull/5313/commits/8975ebce34c75c55cb8f190e7cf5916902140c7e if a dependency is invalid (i.e. column name doesn't exist).

We can have any other warnings at the moment, as we haven't sorted out `@computed` error handling

Reply:
That makes sense! You should add this as a comment

Reply:
Done